pkg/object/gs.go

```diff
@@ -76,9 +76,8 @@ func (g *gs) Create() error {
 	}
 
 	err := g.client.Bucket(g.bucket).Create(ctx, projectID, &storage.BucketAttrs{
-		Name:         g.bucket,
-		StorageClass: "regional",
```

Comment:
https://cloud.google.com/storage/docs/storage-classes#legacy

Reply:
Standard type is better